islandora.install

```diff
@@ -132,3 +132,18 @@ function islandora_update_7001(&$sandbox) {
   $t = get_t();
   return $t("Islandora database updates complete");
 }
+
+/**
+ * Implements hook_update_N().
+ *
+ * Add default permissions for viewing inactive or deleted objects.
+ * Match permissions for viewing all objects.
+ */
+function islandora_update_7002(&$sandbox) {
```

Comment:
I was thinking something like this: 
```php
/**
 * Implements hook_update_N().
 *
 * Notify user of new inactive object permissions.
 */
function islandora_update_7002(&$sandbox) {
  $t = get_t();
  return $t("A new permission 'view inactive or deleted objects' has been added " .
    "to the system, these objects can no longer be viewed by someone without this " . 
    "permission. If you have users that need to view inactive or deleted objects please " .
    "add the new permission to appropriate roles. This is especially important for sites " .
    "using the 'Islandora Simple Workflow' module");
}
```

I am not very good at crafting messages like this sometimes, so maybe this is too much, but its what I was thinking.